python/pycylon/pycylon/data/table.pxd

```diff
@@ -154,5 +167,9 @@ cdef class Table:
         void init(self, const shared_ptr[CTable]& table)
 
         _get_join_ra_response(self, op_name, shared_ptr[CTable] output, CStatus status)
-
         _get_ra_response(self, table, ra_op_name)
+        _get_distributed_tail_response(self, offset, length)
+        _get_distributed_head_response(self, offset, length)
+        _get_distributed_slice_response(self, offset, length)
```

Comment:
looks like the function signature is same, may be you could use a function pointer and maintain DRY.

Reply:
Correct. Initially, it was in a single function. But later it was decided by discussing with Niranda multiple functions by design whether we can create a callback from cython to execute a function pointer in cpp layer. Those are the individual function. But it is a limitation at cython layer that we can't call function pointer.